components/cli/cmd/cellery/cellery.go

```diff
@@ -58,6 +61,31 @@ func newCliCommand() *cobra.Command {
 }
 
 func main() {
+	logFileDirectory := filepath.Join(util.UserHomeDir(), ".cellery", "logs")
+	logFilePath := filepath.Join(logFileDirectory, "cli.log")
+
+	// Creating the log directory if it does not exist
+	err := util.CreateDir(logFileDirectory)
+	if err != nil {
+		log.Fatalf("Failed to create log file: %v", err)
+	}
+
+	// Setting the log output to the log file
+	logFile, err := os.OpenFile(logFilePath, os.O_RDWR|os.O_CREATE|os.O_APPEND, 0666)
+	if logFile != nil {
+		defer func() {
+			err := logFile.Close()
+			if err != nil {
+				log.Fatalf("Failed to close log file: %v", err)
```

Comment:
Do we need this log to be fatal? Since this is just a log file close error, printing an error and continuing should be enough IMO.

Reply:
Changed to a log.Printf()